python/python/lancedb/query.py

```diff
@@ -689,6 +701,31 @@ def phrase_query(self, phrase_query: bool = True) -> LanceFtsQueryBuilder:
         return self
 
     def to_arrow(self) -> pa.Table:
+        tantivy_index_path = self._table._get_fts_index_path()
+        if Path(tantivy_index_path).exists():
+            return self.tantivy_to_arrow()
+
+        # get the scores and doc ids
+        query = self._query
+        if self._phrase_query:
+            raise NotImplementedError("Phrase query is not yet supported")
+        if self._reranker:
+            raise NotImplementedError("Reranking is not yet supported")
```

Comment:
These error messages could be more helpful:
```suggestion
        if self._phrase_query:
            raise NotImplementedError("Phrase query is not yet supported in Lance FTS. Use tantivy-based index instead for now.")
        if self._reranker:
            raise NotImplementedError("Reranking is not yet supported in Lance FTS. Use tantivy-based index instead for now.")
```